test/integration/command-logging-and-monitoring/command_logging_and_monitoring.spec.test.ts

```diff
@@ -12,7 +12,7 @@ import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
 const SKIP = ['A successful unordered bulk write with an unacknowledged write concern'];
 
 describe('Command Logging and Monitoring Spec', function () {
-  describe('Command Monitoring Spec (unified)', () => {
+  describe.only('Command Monitoring Spec (unified)', () => {
```

Comment:
```suggestion
  describe('Command Monitoring Spec (unified)', () => {
```

Reply:
Ah, yeah a stray only, just got rid of it.